app/controllers/vendor_outage_controller.rb

```diff
@@ -16,8 +16,8 @@ def from_idv_phone?
   end
 
   def gpo_letter_available?
-    FeatureManagement.gpo_verification_enabled? &&
-      current_user &&
-      !Idv::GpoMail.new(current_user).rate_limited?
+    return false unless current_user
+    policy = Idv::GpoVerifyByMailPolicy.new(current_user)
+    policy.send_letter_available?
```

Comment:
20 and 21 (and many instances of this across these files) could be inclined to `Idv::GpoVerifyByMailPolicy.new(current_user).send_letter_available?`, but I'm of the school of thought that it's cleaner to do it across two simple lines, and that the memory impact of optimizing for readability is negligible.